src/OrbitGl/ModuleListFuzzer.cpp

```diff
@@ -47,14 +45,10 @@ DEFINE_PROTO_FUZZER(const GetModuleListResponse& module_list) {
   int32_t pid = 1;
   ProcessInfo process_info{};
   process_info.set_pid(pid);
+  ProcessData process = ProcessData(process_info);
 
-  DataManager data_manager{};
-  data_manager.UpdateProcessInfos(std::vector{process_info});
-
-  ProcessData* process = data_manager.GetMutableProcessByPid(pid);
-  CHECK(process != nullptr);
-  process->UpdateModuleInfos(modules);
+  process.UpdateModuleInfos(modules);
 
   ModulesDataView modules_data_view{nullptr};
-  modules_data_view.UpdateModules(process);
+  modules_data_view.UpdateModules(&process);
```

Comment:
this looks suspicious who owns process after that? is it expected to be a temporary? if so why is it not const ref or passed by value?

Reply:
oh, this is fuzzer and modules_data_view is also in the same scope, nevermind